extensions/blocks/subscriptions/shared.scss

```diff
@@ -0,0 +1,69 @@
+.wp-block-jetpack-subscriptions {
+	position: relative;
+
+	.wp-block-jetpack-subscriptions__form,
+	form {
+		display: flex;
+		align-items: start;
+
+		.wp-block-jetpack-subscriptions__textfield .components-text-control__input,
+		.wp-block-jetpack-subscriptions__button,
+		input[type="email"],
+		button {
+			line-height: initial;
+		}
+
+		.wp-block-jetpack-subscriptions__button,
+		button {
+			border-style: solid;
+			border-color: transparent;
+		}
+
+		.wp-block-jetpack-subscriptions__textfield, p#subscribe-email {
```

Comment:
I gather this id and the need for the !important comes from the rendering of the shortcode?

Reply:
The ID comes from the output of the shortcode yes. I'm trying to mess with the shortcode output as little as possible to avoid any breakage on existing sites. The !important I will revisit and see if there's a better way to handle this.

Reply:
!important removed.